### ISSUE TITLE
[Certifier] Allow certification of run, even if it does not exist in RR/OMS API

### ISSUE THREAD

**Issue by nothingface0** [CMSTrackerDPG/certifier#163]
In continuation of #134, when a specific run reconstruction is requested to be cerified but  does not exist in RunRegistry,
Certhelper should allow the user to continue with a manual certification. 

Example: 
> Looks like 346455 was certified for PromptReco and that's available as a Reference, but I can't add the Express.

Reconstruction 346455 Express [does not exist on RunRegistry](https://cmsrunregistry.web.cern.ch/offline/datasets/global?ofbf%5Brules%5D%5B0%5D%5Bfield%5D=run_number&ofbf%5Brules%5D%5B0%5D%5Bvalue%5D=346455&ofbf%5Brules%5D%5B0%5D%5Boperator%5D=%3D&ofbf%5Brules%5D%5B1%5D%5Bfield%5D=name&ofbf%5Brules%5D%5B1%5D%5Bvalue%5D=%25express%25&ofbf%5Brules%5D%5B1%5D%5Boperator%5D=like&ofbf%5Bcombinator%5D=and&ofbf%5Bnot%5D=true), meaning that [Certhelper is not able to create a new certification for this](https://certhelper.web.cern.ch/certify/346455/express/):

![image](https://user-images.githubusercontent.com/8612771/177504631-80cbdee1-eee5-47e4-b258-ea9336ebcf03.png)

This behavior was initially desired, hence [custom exceptions were created](https://github.com/CMSTrackerDPG/certifier/blob/a982b4b66234113e81cbdc4106ef708637dd1d5d/certifier/views.py#L132), to differentiate such cases.

**TL;DR:** When a run reconstruction is missing from RunRegistry, CertHelper should be able to continue in the same manner that it does when `ConnectionError`s are raised when connecting to RunRegistry or OMS.
